rotkehlchen/tests/unit/decoders/test_extrafi.py

```diff
@@ -420,9 +420,144 @@ def test_farm_investment(
             asset=Asset('eip155:10/erc20:0x1e925De1c68ef83bD98eE3E130eF14a50309C01B'),
             balance=Balance(amount=FVal(deposited_amount)),
             location_label=optimism_accounts[0],
-            notes=f'Deposit {deposited_amount} EXA in Extrafi',
+            notes=f'Deposit {deposited_amount} EXA in Extrafi EXA-WETH farm',
             counterparty=CPT_EXTRAFI,
             extra_data={'vault_id': 70, 'vault_position': 511},
             address=string_to_evm_address('0x9558FF42E95dcA076A8DEB67c8FF8B86f52b2f8C'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=69,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.GENERATE_DEBT,
+            asset=Asset('eip155:10/erc20:0x4200000000000000000000000000000000000006'),
+            balance=Balance(amount=FVal(borrow_amount)),
+            location_label=optimism_accounts[0],
+            notes=f'Borrow and deposit {borrow_amount} WETH in Extrafi EXA-WETH farm',
+            counterparty=CPT_EXTRAFI,
+            address=string_to_evm_address('0xf9cFB8a62f50e10AdDE5Aa888B44cF01C5957055'),
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('base_accounts', [['0x007183900fBbe3e7815b278074a49B8C7319EDba']])
+def test_new_farm_borrow_position_on_base(base_inquirer, base_accounts):
+    evmhash = deserialize_evm_tx_hash('0xb9479f2e21100ddbba10395d76abb2fb4e151b2142ba90c91151a10fcb5cfbc7')  # noqa: E501
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=base_inquirer, tx_hash=evmhash)
+    timestamp, borrow_amount, gas_fees = TimestampMS(1725972969000), '4025.689364', '0.000006907473477667'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_fees)),
+            location_label=base_accounts[0],
+            notes=f'Burned {gas_fees} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=evmhash,
+            sequence_index=440,
+            timestamp=timestamp,
+            location=Location.BASE,
+            event_type=HistoryEventType.RECEIVE,
+            event_subtype=HistoryEventSubType.GENERATE_DEBT,
+            asset=Asset('eip155:8453/erc20:0x833589fCD6eDb6E08f4c7C32D4f71b54bdA02913'),
+            balance=Balance(amount=FVal(borrow_amount)),
+            location_label=base_accounts[0],
+            notes=f'Borrow and deposit {borrow_amount} USDC in Extrafi USDz-USDC farm',
```

Comment:
Hmmm wait I am confused here. What does borrow and deposit mean here? These are two different things. In a single event you either borrow something from somewhere or you deposit it to somewhere. What's happening here? It's not clear from etherscan